.circleci/config.yml

```diff
@@ -453,13 +453,15 @@ jobs:
             cd ..
             yarn detox clean-framework-cache
             yarn detox build-framework-cache
+            yarn detox build --configuration ios.sim.debug
```

Comment:
The build is already done [in the previous step](https://github.com/pagopa/io-app/pull/3482/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R441).
Also, I'd rather test the `release` config because it better reflects what we send to the store.

Reply:
Removed the unnecessary step in https://github.com/pagopa/io-app/pull/3482/commits/17ac2a80119c8c25ec3e9475dbcfe1fe1e53e59f.

As for using the release configuration, I had tried it before but didn't get a positive run. 
I have tried again now and this is the result:
https://app.circleci.com/pipelines/github/pagopa/io-app/16172/workflows/4843c73e-fa37-42f1-a0a4-75cd78143c2a/jobs/98677/artifacts

I think there is something that doesn't work in the CI with the release configuration, maybe we could investigate the problem in a second time and keep using the debug configuration at the moment, since it works correctly. What you think?